plugins/woocommerce/includes/class-wc-post-types.php

```diff
@@ -379,6 +379,9 @@ public static function register_post_types() {
 									array(
 										'title'       => __( 'Basic details', 'woocommerce' ),
 										'description' => __( 'This info will be displayed on the product page, category pages, social media, and search results.', 'woocommerce' ),
+										'icon'        => array(
+											'src' => '<svg xmlns="http://www.w3.org/2000/svg" view-box="0 0 24 24"><path fill-rule="evenodd" d="M5 5.5h14a.5.5 0 01.5.5v1.5a.5.5 0 01-.5.5H5a.5.5 0 01-.5-.5V6a.5.5 0 01.5-.5zM4 9.232A2 2 0 013 7.5V6a2 2 0 012-2h14a2 2 0 012 2v1.5a2 2 0 01-1 1.732V18a2 2 0 01-2 2H6a2 2 0 01-2-2V9.232zm1.5.268V18a.5.5 0 00.5.5h12a.5.5 0 00.5-.5V9.5h-13z" clip-rule="evenodd" /></svg>',
+										),
```

Comment:
Did you take these svg's from Figma or from `@wordpress/icons`?
Instead of pasting the entire SVG in here, could you potentially add them to one of the folders and could you use `file_get_contents` maybe to add it here?